src/date.c

```diff
@@ -1563,6 +1566,21 @@ static void ctimestampFunc(
 ** and strftime(). The format string to pass to strftime() is supplied
 ** as the user-data for the function.
 */
+
```

Comment:
Is SQLITE_OMIT_DATETIME_FUNCS defined?  If so, why?  Is this something missing from the kernel?

Reply:
SQLITE_OMIT_DATETIME_FUNCS is defined to 1. Regardless of that, sqlite3 assumes that the gmtime() function exists, which is in the <time.h> library file. This does not exist in the kernel, so I have excluded the code which exists when SQLITE_OMIT_DATETIME_FUNCS is defined to 1, and which is meant to replace the code which is executed when SQLITE_OMIT_DATETIME_FUNCS is set to 0.